bagua/script/baguarun.py

```diff
@@ -87,22 +89,42 @@ def pssh_bagua_launch(
             }
         )
 
-    client = ParallelSSHClient(host_list, port=ssh_port)
+    enable_host_logger()
+    client = ParallelSSHClient(
+        host_list, host_config=host_config, pool_size=10 + len(host_list)
+    )
     output = client.run_command(
         "%(cmd)s",
         host_args=host_args,
         shell="bash -xc",
         use_pty=True,  # The key configuration of process safe exit
     )
-    host_out = output[0]
-    for line in host_out.stdout:
-        print(line, flush=True)
+
+    def do_output(host_out):
+        for line in host_out.stdout:
+            pass
+
+    cmds = [client.pool.spawn(do_output, host_out) for host_out in output]
+    gevent.joinall(cmds)
+
+    client.join()
 
 
 def parse_args():
     parser = argparse.ArgumentParser(description="Distributed bagua launch")
-    parser.add_argument("--host_list", default=None)
-    parser.add_argument("--ssh_port", type=int, default=None)
+    parser.add_argument(
+        "--host_list", default=None,
+        help="List of target nodes, baguarun logs in to `host` via ssh. "
+        "Specify a list of nodes through host_list, which is a "
+        "comma-separated dns or ip. If the ssh port of the node is the "
+        "same, you can specify it by --ssh_port, if it is different, you "
+        "need to add: SSH_PORT after dns or ip.\n"
```

Comment:
add `:SSH_PORT`?

Better to give an example for easier understanding. `host1:port1,host2:port2,...`?

Reply:
Also after `dns` is not correct technically. should be after domain name

Reply:
DONE